packages/tooltip/README.md

```diff
@@ -70,7 +70,7 @@ import Tooltip from '@leafygreen-ui/tooltip';
 | `enabled`         | `boolean`                                | Enables Tooltip to trigger based on the event specified by `triggerEvent`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                     | `true`                 |
 | `usePortal`       | `boolean`                                | Determines if the Tooltip will be rendered within a portal.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    | `true`                 |
-| `portalContainer` | `HTMLElement` \| `null`                  | Sets the container used for the popover's portal.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |                        |
+| `portalContainer` | `HTMLElement` \| `null`                  | Sets the container used for the popover's portal. NOTE: If using a `scrollContainer` make sure that the `portalContainer` is contained within the `scrollContainer`. E.g, passing the same refrence to `scrollContainer` and `portalContainer`.                                                                                                                                                                                                                                                                                                |                        |
```

Comment:
Future consideration🔮:  We should really have this prop documentation read off the same config so we don't have to change it multiple times